src/syncano.js

```diff
@@ -1,15 +1,30 @@
 import stampit from 'stampit';
 import models from './models';
+import _ from 'lodash';
 
 /**
  * Main Syncano object.
  * @type {Syncano}
  */
-const Syncano = stampit();
-
-models.Instance.please().list().then((response) => {
-    response.objects[0].delete();
-    response.objects[0].save();
-})
+const Syncano = stampit()
+  .init(({instance}) => {
+    _.forEach(models, (model, name) => {
+      instance[name] = stampit().compose(model).setBaseObject(this)();
```

Comment:
If you don't use the `()` then syncano.Instance will return the factory, so you'll need to use it as `syncano.Instance()` ;)